src/status_im/chat/events.cljs

```diff
@@ -165,14 +181,20 @@
 (handlers/register-handler-fx
   :initialize-chats
   [(re-frame/inject-cofx :all-stored-chats)
```

Comment:
it would be good to have macro or function something like `(inject-cofxs [:cofx1 :cofx2 :cofx3])` but not related to this PR

Reply:
like `(map re-frame/inject-cofx [:cofx1 :cofx2 :cofx3])`? :)

Reply:
mapv then to keep a vector